README.md

```diff
@@ -346,6 +346,20 @@ await when(() => {
 })
 ```
 
+Returns a thennable function that can both be used as a callback and
+awaited on directly. For example, testing frameworks that support
+async tests work well with convergent assertions.
+
+``` javascript
+test('will eventually become bar within 1s', when(() => {
+  expect(foo).to.equal('bar');
+}, 1000));
```

Comment:
Do you have to specify the timeout, or if you leave it off it'll use the default 2s?

Reply:
Defaults to 2s. The note in the readme is right under the section that specifies what the defaults are (can't see from the diff)